dds_cli/base.py

```diff
@@ -146,23 +145,12 @@ def __get_key(self, private: bool = False):
         """Get public key for project."""
         key_type = "private" if private else "public"
         # Get key from API
-        try:
-            response = requests.get(
-                DDSEndpoint.PROJ_PRIVATE if private else DDSEndpoint.PROJ_PUBLIC,
-                params={"project": self.project},
-                headers=self.token,
-                timeout=DDSEndpoint.TIMEOUT,
-            )
-        except requests.exceptions.RequestException as err:
-            LOG.fatal(str(err))
-            raise SystemExit(
-                "Failed to get cloud information"
-                + (
-                    ": The database seems to be down."
-                    if isinstance(err, requests.exceptions.ConnectionError)
-                    else "."
-                )
-            ) from err
+        response = utils.request_get.get(
+            DDSEndpoint.PROJ_PRIVATE if private else DDSEndpoint.PROJ_PUBLIC,
+            params={"project": self.project},
+            headers=self.token,
+            error_message="Failed to get cloud information: The database seems to be down.",
+        )
 
         if not response.ok:
```

Comment:
the response check and getting the json are handled inside the request_get and will fail here anyways